### PR TITLE
added logic and styling Task 5, back button etc

### PR DESCRIPTION
**Pull request by Cloud-Monkey**
Task 5. Have added the single review by ID view as a link from the reviews in the list, also added back button and review body on full view, basic uniformed styles :)  pls and thx

### CODE REVIEW

File: src/components/ReviewFullPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        console.log(error);
+        setLoading(false);
+      });
+  }, [reviewId]);
+
+  if (loading || review === null) {
+    return <div>Loading...</div>;
+  }
+  return (
+    <div>
```
Comment (itskatherine) [inline src/components/ReviewFullPage/index.jsx:30]: To save adding another `<div>` to the mix, while still being able to wrap the JSX you're returning in one element, I'd suggest using a React fragment, i.e <> </> instead, which will disappear when you actually render the page ✨

File: src/utils/apis.js [hunk 2]
```diff
@@ -9,3 +9,10 @@ export const getReviews = () => {
     return data.review;
   });
 };
+
+export const getReviewById = (reviewId) => {
+  return ncGamesApi.get(`/reviews/${reviewId}`).then(({ data }) => {
+    console.log(data.review[0]);
```
Comment (itskatherine) [inline src/utils/apis.js:15]: Console logs begone 🗑️

File: src/components/ReviewFullPage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import ReviewCard from "../ReviewCard";
+import { getReviewById } from "../../utils/apis";
+import "./styles.css";
```
Comment (itskatherine) [inline src/components/ReviewFullPage/index.jsx:5]: CSS is looking good, Dave! 🎨

**Review by itskatherine — APPROVED** [review]
Nice work Dave, CSS is looking good and responsive ✅ Going forwards though, try where possible to use more semantic elements, rather than lots of divs.
It would be good if the header title "NC Games!" were also a link to the homepage
Have a look over my comments, then you're good to merge 👍

File: src/components/ReviewFullPage/index.jsx (hunk 1 above)
Comment (Cloud-Monkey) [reply src/components/ReviewFullPage/index.jsx:30]: ah fragments, yeah Ive used one before, can't remember where tho' lol :)

File: src/utils/apis.js (hunk 2 above)
Comment (Cloud-Monkey) [reply src/utils/apis.js:15]: haha ooops

File: src/components/ReviewFullPage/index.jsx (hunk 3 above)
Comment (Cloud-Monkey) [reply src/components/ReviewFullPage/index.jsx:5]: Spent a little bit of time trying to get eveything neat for now, thx ;)

**Pull request merged**